index.html

```diff
@@ -61,27 +61,323 @@
+                  <i class="bi bi-cart"></i>
+                </div>
+              </div>
+            </div>
+          </section>
+          <section class="component">
+            <div class="card__container">
+              <div class="card-header">
+                <img src="/images/drools-image.png" alt="" class="card-image" />
+                <span class="card-badge">20% OFF</span>
+              </div>
+              <div class="card-details">
+                <h2 class="detail-heading">Drools for Dogs</h2>
+                <p class="detail-subheading">Adult 1.2 Kg</p>
+              </div>
+              <div class="card-footer">
+                <div class="price-segment">
+                  <span class="item-price">&#x20b9; 250</span>
+                  <span class="item-price dashed">&#x20b9; 350</span>
+                </div>
+                <div class="card-icons">
+                  <i class="bi bi-heart"></i>
+                  <i class="bi bi-cart"></i>
+                </div>
+              </div>
+            </div>
+          </section>
+          <section class="component">
+            <div class="card__container">
+              <div class="card-header">
+                <img
+                  src="/images/pedigree-image.png"
+                  alt=""
+                  class="card-image"
+                />
+                <span class="card-badge">20% OFF</span>
+              </div>
+              <div class="card-details">
+                <h2 class="detail-heading">Pedigree for Dogs</h2>
+                <p class="detail-subheading">Adult 1.2 Kg</p>
+              </div>
+              <div class="card-footer">
+                <div class="price-segment">
+                  <span class="item-price">&#x20b9; 250</span>
+                  <span class="item-price dashed">&#x20b9; 350</span>
+                </div>
+                <div class="card-icons">
+                  <i class="bi bi-heart"></i>
+                  <i class="bi bi-cart"></i>
+                </div>
+              </div>
+            </div>
+          </section>
         </div>
       </main>
+
+      <footer class="footer">
+        <div class="footer-container">
+          <div class="social-media">
+            <div class="logo">
+              <a href="#"
+                ><img
+                  src="/images/PetmateLogo.png"
+                  alt="Petmate-logo"
+                  class="header__logo"
+              /></a>
+            </div>
+            <div class="footer-info">
+              <h3 class="footer-h3">One place shop to all pet lovers</h3>
+              <h3 class="footer-h3">@2022, All rights reserved</h3>
+              <h3 class="footer-h3">Made with love by Arjun</h3>
+            </div>
+            <div class="media-links">
+              <a href=""><i class="fab fa-twitter"></i></a>
+              <a href=""><i class="fab fa-linkedin"></i></a>
+              <a href=""><i class="fab fa-github"></i></a>
```

Comment:
Links to media links are missing. Also can add target="_blank" so that external link open in new tab